04_documentation/examples/reStructuredText-example.rst

```diff
@@ -0,0 +1,71 @@
+==================
+Top-level headline
+==================
+
+Overlining is optional. One can use other characters for the headline marking such as ``-`` or ``*``. There is no strict hierarchy of these characters defined to mark sections, subsections etc. The actual hierarchy is determined by the order of headings. However, the Python style guide proposes ``#`` > ``*`` > ``=`` and so on with each character referring to a certain type of structure (part, chapter, section...)
+
+More information can be found in the `Python documentation style guide <https://devguide.python.org/documenting/#sections>`_
+
+-------------------------------
+This is the second-level header
+-------------------------------
+
+Hello there.
+
+*This text is emphasized (italics)*
+
+**This text is heavily emphasized (bold)**
+
+``This is monofont text``
+
+We can also have nice lists
+
+* Item 1
+* Item 2
+
+which can be numbered as well
+
+1. Item 1
+2. Item 2
+
+   1. Item 2a
+   2. Item 2b
+
+As one can see we can also have sublists. In contrast to markdown there has to be a line break before the sublist starts.
+
+Code uses backticks. This can be code/variable names in text that ``appear`` in a monospace font. Code blocks start and end with triple backticks and often take a specifier of the presented language.
```

Comment:
I guess the last sentence is a copy-paste problem.

Reply:
Yes. It is fixed now.